gerritcheck/check.py

```diff
@@ -133,6 +133,12 @@ def cppcheck_on_files(files, commit):
             review["labels"] = {"Code-Review": -1}
             return json.dumps(review)
 
+    # Check the return code only just now as cppcheck might still have returned
+    # some valid comments.
+    if rc != 0:
```

Comment:
Hmm, I guess there are cases when cppcheck returns non-zero and the developer is able to fix that. I'm e.g. thinking about path issues in cross-platform development. On Windows, the path length is usually limited to 256 chars. Suppose some paths in your repo are longer than that, then cppcheck would not be able to access them and return non-zero, yet the developer could fix it by moving these files to a shorter path within the repo. Just as an example.